tck/src/main/java/org/eclipse/microprofile/openapi/apps/airlines/resources/ZepplinResource.java

```diff
@@ -36,7 +40,30 @@ public class ZepplinResource {
     @Operation(summary = "Deprecate outdated airship technology", operationId = "deprecateZepplin")
     @Produces("text/plain")
     @SecurityRequirement(name = "mutualTLSScheme", scopes = "zepplinScope")
-    public Response deprecateZepplin() {
+    public Response deprecateZepplin(@RequestBody(description = "Something about a zepplin.",
+                                                  content = @Content(mediaType = "application/json")) String string) {
+        return Response.ok().build();
+    }
+
+    @HEAD
+    @Path("{id}")
+    @APIResponse(responseCode = "200", description = "Review deleted")
+    @APIResponse(responseCode = "404", description = "Review not found")
+    @Operation(summary = "Deprecate outdated airship technology", operationId = "deprecateZepplin")
+    @Produces("text/plain")
+    @SecurityRequirement(name = "mutualTLSScheme", scopes = "zepplinScope")
+    public Response headZepplin(@RequestBody(ref = "#/paths/~1zepplins~1{id}/delete/requestBody") String string) {
+        return Response.ok().build();
+    }
```

Comment:
The test looks good, but I'm slightly concerned that Jakarta REST may not be required to support this. Will take a look.

Reply:
I can't _see_ anything in the Jakarta REST spec that says either way whether this is permitted. You're welcome to also have a look, but unless we're sure that this is required to be supported, I think we should test this in `StaticDocumentTest` or `ModelReaderAppTest`.

We only want to test that their MP OpenAPI implementation can document this kind of method, not that their Jakarta REST implementation allows it.